can-define.js

```diff
@@ -719,6 +719,14 @@ define.setup = function(props, sealed) {
 };
 define.replaceWith = replaceWith;
 define.eventsProto = eventsProto;
+define.addBaseEvents = function(objPrototype){
+	objPrototype._baseEvents = objPrototype.addEventListener &&
```

Comment:
I don't follow where the problem is, but ok, how else would you recommend doing this?